app/components/rocketchat.js

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
```

Comment:
lets move this to `inappchat/inappchat.js`

Reply:
done